accelerators/stratus_hls/common/hls/project.tcl

```diff
@@ -45,11 +48,14 @@ if {[lsearch $fpga_techs $TECH] >= 0} {
     set TECH_IS_XILINX 1
 
 }
-if {[lsearch $asic_techs $TECH] >= 0} {
-    set_attr verilog_files "$ESP_ROOT/rtl/sim/$TECH/verilog/*v $ESP_ROOT/rtl/techmap/$TECH/mem/*v"
+if {[lsearch $asic_techs $TECH_TYPE] >= 0} {
```

Comment:
TECH_TYPE is always "asic" now? and then TECH is the particular technology? Because I see we are using TECH to set the frequency/input delays

Reply:
Yes. For asic flow, TECH_TYPE is always asic. This make ESP repo with only "asic" files instead of having files foe each asic technology.
TECH is for the particular technology to keep similar structure inside esp/tech folder. Therefore, for a given asic technology, every HLS generated accelerator will have its specific technology folder and would avoid future mistakes in a project with tight deadline.